### ISSUE TITLE
Auto Completion missing available fields after SELECT

### ISSUE THREAD

**Issue by jkmuka** [forcedotcom/salesforcedx-vscode#1886]
The auto-completion works on the objects after the FROM but does not work on the fields after the SELECT

### Steps To Reproduce:

1.  Ctrl+space, select "SELECT {Field}  FROM {SObject}" snippet and hit enter
1.  Use down error to select Account and hit enter
1.  Place cursor anywhere after SELECT but before FROM, start typing or press Ctrl+space

### Expected result
Expected to see list of fields

### Actual result
No fields shown, only additional commands

### Additional information

**VS Code Version**: 1.14.1

**SFDX CLI Version**: sfdx-cli/7.36.0-b1c4e9c7c8 win32-x64 node-v10.15.3

**OS and version**: Windows10
![image](https://user-images.githubusercontent.com/6767449/72667577-1aec1a00-39eb-11ea-87e6-cf2e9161ce6c.png)

**Comment by ntotten** [comment]
Have you run the refresh sobject definitions command?

**Comment by jkmuka** [comment]
Yes. Also the the fields are auto-suggested in the WHERE clause so I know that there references are there

![image](https://user-images.githubusercontent.com/6767449/72667622-dad96700-39eb-11ea-85df-1cc2a803a758.png)
